src/components/tx/SignOrExecuteForm/SafenetTxChecks.tsx

```diff
@@ -1,16 +1,15 @@
-import { type ReactElement, useContext } from 'react'
+import { type ReactElement } from 'react'
 import { SafenetTxSimulation } from '@/components/tx/security/safenet'
 import TxCard from '@/components/tx-flow/common/TxCard'
-import { SafeTxContext } from '@/components/tx-flow/SafeTxProvider'
 import useIsSafenetEnabled from '@/hooks/useIsSafenetEnabled'
 import { Typography } from '@mui/material'
 import useChainId from '@/hooks/useChainId'
 import useSafeAddress from '@/hooks/useSafeAddress'
+import type { SafeTransaction } from '@safe-global/safe-core-sdk-types'
 
-const SafenetTxChecks = (): ReactElement | null => {
+const SafenetTxChecks = ({ safeTx }: { safeTx?: SafeTransaction }): ReactElement | null => {
```

Comment:
why is the `safeTx`param optional? can it execute any checks if safeTx is undefined ?

Reply:
I kept the same data definition as before 
<img width="392" alt="image" src="https://github.com/user-attachments/assets/91e4d97f-9c85-474a-af3a-141bb7d21c0e">

Reply:
But this is the context definition, it can be empty or contain a transaction. I don't think they're related

Reply:
yeah you're right, let's not make it optional